src/main/java/com/starbucks/controllers/RestController.java

```diff
@@ -1,26 +1,38 @@
 package com.starbucks.controllers;
 
 import com.starbucks.model.Order;
-import com.starbucks.repositary.OrderRepositary;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
+import com.starbucks.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
 
 @org.springframework.web.bind.annotation.RestController
-@RequestMapping("/order")
 public class RestController {
 
-    private final OrderRepositary orderRepositary;
+    private final OrderService orderService;
 
-    public RestController(OrderRepositary orderRepositary) {
-        this.orderRepositary = orderRepositary;
+    public RestController(OrderService orderService) {
+        this.orderService = orderService;
     }
 
-    @PostMapping
+    @PostMapping("/orders")
     public void storeOrder(@RequestBody Order order) {
-        order.toString ( );
-        orderRepositary.saveOrder ( order );
+        orderService.saveOrder ( order );
```

Comment:
don't use your own formatting style, use autoformat ctrl+alt+L (ctrl+alt+O for imports autoformat). It's not a question of your taste it's a precise convention. https://www.oracle.com/technetwork/java/codeconventions-150003.pdf - proof )